samples/net/rpl-node/README.rst

```diff
@@ -0,0 +1,72 @@
+.. _rpl-node-sample:
+
+RPL node
+###########
+
+Overview
+********
+
+A simple RPL node showing how to join in RPL mesh network.
+
+This demo assumes that the platform of choice has networking support,
+some adjustments to the configuration may be needed.
+
+The sample will listen for RPL multicast messages and joins with RPL
+Border Router node in DAG network. It exposes few resources through
+CoAP server role.
+
+The sample exports the following resources:
```

Comment:
... and change this to read:

     The sample exports the following resources through the CoAP server role: